src/main/java/com/openshift/internal/restclient/model/Pod.java

```diff
@@ -61,10 +76,35 @@ public String getHost() {
 		}
 		return images;
 	}
+	
+	/**
+	 * The logic of the method is a copied from 'podStatus' function of
+	 * [app/scripts/filters/resources.js] of [openshift/origin-web-console]
+	 */
 
 	@Override
 	public String getStatus() {
-		return asString(POD_STATUS);
+		if (has(POD_DELETION_TIMESTAMP)) {
+			return "Terminating";
```

Comment:
I also see us adding a state here, that doesnt exist in openshift/origin. Shouldnt we file this issue?